api_app/analyzers_manager/migrations/0094_analyzer_config_malprobsearch.py

```diff
@@ -0,0 +1,123 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": {
+            "minute": "0",
+            "hour": "0",
+            "day_of_week": "*",
+            "day_of_month": "*",
+            "month_of_year": "*",
+        },
+        "update_schedule": None,
+        "module": "malprob.MalprobSearch",
+        "base_path": "api_app.analyzers_manager.observable_analyzers",
+    },
+    "name": "MalprobSearch",
+    "description": "[Malprob](https://malprob.io/) is a leading malware detection and identification service, powered by cutting-edge AI technology.",
+    "disabled": False,
+    "soft_time_limit": 10,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "observable",
+    "docker_based": False,
+    "maximum_tlp": "AMBER",
+    "observable_supported": ["hash"],
+    "supported_filetypes": [],
+    "run_hash": False,
+    "run_hash_type": "",
+    "not_supported_filetypes": [],
+    "model": "analyzers_manager.AnalyzerConfig",
+}
+
+params = []
```

Comment:
The api key parameter is missing

Reply:
the search doesn't require api key

Reply:
![image](https://github.com/intelowlproject/IntelOwl/assets/30528879/53b91c33-c202-454e-bc22-c5dc66df69ed)
Then this should be changed :P